lib/deployinator/app.rb

```diff
@@ -184,8 +184,8 @@ def github_diff_url(params)
         Signal.trap("HUP") { exit }
         if Deployinator.log_file?
           log = File.new(Deployinator.log_file, "a")
-          $stdout.reopen(log)
-          $stderr.reopen(log)
+          $stdout.sync = true
+          $stderr.sync = true
```

Comment:
I don't think this will actually log to the file anymore. I'll test it out but I think we might need to set sync to true and redefine $stdout.write to write to our logfile as well. Another solution would be to replace all the "puts" in our code with a different log method. STDERR can still be redirected without angering passenger afaict